Makefile

```diff
@@ -36,16 +36,27 @@ build: build-assets
 
 HUGO_SERVE := hugo serve --buildDrafts --buildFuture --ignoreVendorPaths="github.com/pulumi/registry/**/*" | grep -v -e 'WARN .* REF_NOT_FOUND'
 
-# bin/resourcedocsgen is considered to depend on all files in the tools/resourcedocsgen directory.
-#
-# Caveat: This does not account for `replace` directives in `go.mod`.
-bin/resourcedocsgen: $(shell find tools/resourcedocsgen | sed 's@ @\\ @g')
+HELPMAKEGO_VERSION := v0.1.0
+HELPMAKEGO := bin/${HELPMAKEGO_VERSION}/helpmakego
+${HELPMAKEGO}:
```

Comment:
I think this should depend on `bin` too

Reply:
It doesn't need to. `go` will create directories as needed.